src/github_amazon_bot/core.clj

```diff
@@ -16,20 +17,25 @@
         num-chars (count first-occurence)]
     (> num-chars 0)))
 
+(defn get-items-to-watch
+  "Get list of items to watch from a file"
+  []
+  (let [file-contents (slurp "shopping_list.json")]
+    (cc/parse-string file-contents true)))
+
 (defn -main
   "Main function"
   []
-  (let [ipads [{:url "https://www.amazon.in/dp/B0864JNFB8/" :name "iPad Pro 12.9"}
-               {:url "https://www.amazon.in/Apple-iPad-11-inch-Wi-Fi-128GB/dp/B0864JKY83" :name "iPad Pro 11"}]
+  (let [items (get-items-to-watch)
         text-to-search "currently unavailable"
         token (env :telegram-token)
         chat-id (env :telegram-chat-id)]
-    (doseq [ipad ipads :let [name (ipad :name)
-                             url (ipad :url)
+    (doseq [item items :let [name (item :name)
+                             url (item :url)
                              body (get-site-body url)
-                             ipad-is-available (not (is-text-present? text-to-search body))]]
-      (when ipad-is-available
+                             item-is-available (not (is-text-present? text-to-search body))]]
+      (when item-is-available
         (try
           (api/send-text token chat-id (str name " is available! Please buy it from " url))
-          (catch Exception))))))
+          (catch Exception ex))))))
```

Comment:
I see that you've fixed it already :)

Reply:
Linter was showing a warning for unused var. Not sure how to fix that, so leaving it as is.